src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServices.cs

```diff
@@ -10,12 +12,20 @@ public static class NuGetServices
     {
         private const string NuGetSolutionServiceName = "NuGetSolutionService";
         private const string NuGetSolutionServiceVersion = "1.0.0";
+        private const string SourceProviderServiceName = "NuGet.SourceProviderService";
```

Comment:
>   [](start = 0, length = 2)

let's have the name (period or not) be consistent here.

Reply:
If you look at VS's RPC contracts assembly, all services use a long `Microsoft.VisualStudio.<component>.<service>` naming convention, similar to what I used in https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGetServices.cs#L13.  Assuming we want to follow conventions :)